File: GTTS/INL.TrainingService.Models/SaveTrainingEventRoster_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventRoster_Result : BaseResult, ISaveTrainingEventRoster_Result
    {
        public long TrainingEventID { get; set; }
        public long? TrainingEventGroupID { get; set; }
        public List<ITrainingEventRoster_Item> Students { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-roster_result.ts


import { ITrainingEventRoster_Item } from './itraining-event-roster_item';
import { ISaveTrainingEventRoster_Result } from './isave-training-event-roster_result';

export class SaveTrainingEventRoster_Result implements ISaveTrainingEventRoster_Result {
  
	public TrainingEventID: number = 0;
	public TrainingEventGroupID?: number;
	public Students?: ITrainingEventRoster_Item[];
  
	public ErrorMessages: string[] = null;
}






